phoenix4-connectors-assembly/pom.xml

```diff
@@ -0,0 +1,224 @@
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>ln</executable>
+                            <workingDirectory>${project.basedir}/../phoenix-pig-base/phoenix4-pig/target</workingDirectory>
+                            <arguments>
+                                <argument>-fnsv</argument>
+                                <argument>
+                                    phoenix4-pig-${project.version}.jar
+                                </argument>
+                                <argument>
+                                    phoenix4-pig.jar
+                                </argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>spark without version</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>ln</executable>
+                            <workingDirectory>${project.basedir}/../phoenix-spark-base/phoenix4-spark/target</workingDirectory>
+                            <arguments>
+                                <argument>-fnsv</argument>
+                                <argument>
+                                    phoenix4-spark-${project.version}.jar
+                                </argument>
+                                <argument>
+                                    phoenix4-spark.jar
+                                </argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>kafka without version</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>ln</executable>
+                            <workingDirectory>${project.basedir}/../phoenix-kafka-base/phoenix4-kafka/target</workingDirectory>
+                            <arguments>
+                                <argument>-fnsv</argument>
+                                <argument>
+                                    phoenix4-kafka-${project.version}.jar
+                                </argument>
+                                <argument>
+                                    phoenix4-kafka.jar
+                                </argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
```

Comment:
The directory created by this section is not copied into the assembly file.
Luckily, since if it was, there would be nothing to serve the artifacts from there.

Please remove this (from both assemblies), or explain why this should be kept.